tools/wptserve/wptserve/pipes.py

```diff
@@ -320,6 +320,8 @@ def __init__(self, params):
 
     def __getitem__(self, key):
         try:
+            if isinstance(key, text_type):
+                key = key.encode('iso-8859-1')
```

Comment:
It seems worth documenting why we're encoding here, and why to iso-8859-1.

Reply:
We should probably document it somewhere (in a central place), but probably not here. We will end up having `iso-8859-1` littered almost everywhere in wptserve and we probably don't want to repeat the lengthy comment. Right now, `request.py` has some comments about this.